src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$card-during-anim: all 300ms ease-out;
+$header-during-anim: all 300ms ease-out;
```

Comment:
don't use `all` in your transitions, better specify all properties to make them refactorable in future